src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from "react-router-dom";
+
+export default function commentCard(props) {
+  const { comment } = props;
+
+  return (
+    <li className="comment-card" comment={comment}>
+      {comment.created_at.slice(0, 10)}
+      <br />
+      <p>{comment.body}</p>
+      <p>Votes: {comment.votes}</p>
+    </li>
+  );
+}
```

Comment:
Looks good Neil, well done breaking this down into a component.

Remember a component should start with a capital letter, so it should be `CommentCard`